src/main.rs

```diff
@@ -15,11 +21,24 @@ fn parse_command(msg: PrivmsgMessage) {
         Some("!Bazylia") => println!("{}", include_str!("../assets/bazylia.txt")),    
         Some("!Zoya") => println!("{}", include_str!("../assets/zoya.txt")),
         Some("!Discord") => println!("https://discord.gg/UyrsFX7N"),
+        Some("!code") =>  format_code(&content),
         _ => {},
     }
 }
 
 
+// Takes code after !code and writes into scrap.rs then formats with fmt - requires rustfmt installed (apt-get)
+fn format_code(message:&str) {
```

Comment:
it seems we don't need the scrap.rs file if we have this? I guess it would be nicer to have a scrap.rs file so we can e.g. test that separately

Reply:
See comment one